kasa/iotprotocol.py

```diff
@@ -0,0 +1,106 @@
+"""Module for the IOT legacy IOT KASA protocol."""
+import asyncio
+import logging
+from typing import Dict, Optional, Union
+
+import httpx
+
+from .credentials import Credentials
+from .exceptions import AuthenticationException, SmartDeviceException
+from .json import dumps as json_dumps
+from .klaptransport import TPLinkKlapTransport
+from .protocol import TPLinkProtocol, TPLinkTransport
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class TPLinkIotProtocol(TPLinkProtocol):
```

Comment:
```suggestion
class IotProtocol(TPLinkProtocol):
```
We probably want to have some documentation (or at least docstrings) explaining what different protocols mean in the context of the lib, considering IoT is rather generic term :-)

Reply:
Done